de/software/praktikumsplaner.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Praktikumsplaner
+developer: LHM
+code: https://github.com/it-at-m/Praktikumsplaner
+codelinktext: github.com/it-at-m/praktikumsplaner
+licensingmodel: open source
+logo: /inhouse/praktikumsplaner_logo.png
+license: MIT
+tags:
+  - application
+  - eigenentwicklung
+  - start2023
+sortingPriority: 3
+---
+Der __Praktikumsplaner__ bietet eine Möglichkeit Nachwuchskräfte und ihre Praktikumsstellen während der Ausbildung zu organisieren.
+
+---
+
+## Über das Projekt
+
+Der Praktikumsplaner, ein Projekt, das in Zusammenarbeit zwischen dem Competence Center of Software-Engineering (CCSE) und dem InnovationsLab ins Leben gerufen wurde, zielt darauf ab, die Ausbildungsleitung bei der Zuweisung von Nachwuchskräften zu unterstützen.
+Er bietet den Ausbilderinnen und Ausbildern die Möglichkeit ihre verfügbaren Praktikumsplätze an die Ausbildungsleitung zu melden.
+Diese kann dann mithilfe der Anwendung die Nachwuchskräfte gezielt an die gemeldeten Praktikumsplätze per Drag-and-Drop-Funktionalität zuteilen.
+Hierbei hilft der Praktikumsplaner einen guten Überblick zu behalten, um die Nachwuchskräfte an möglichst passende Stellen zuzuweisen.
+
+![Screenshot der Zuweisungseite des Praktikumsplaners mit Beispieldaten](/public/inhouse/Screenshot_Praktikumsplaner.png)
```

Comment:
`/public/` is not used hier (see [zeitmanagementsystem.md](https://github.com/it-at-m/opensource.muenchen.de/blob/main/software/zeitmanagementsystem.md?plain=1#L35))